man/systemd.generator.xml

```diff
@@ -163,6 +163,15 @@
         <xi:include href="version-info.xml" xpointer="v251"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>$SYSTEMD_SOFT_REBOOTS_COUNT</varname></term>
+
+        <listitem><para>If the system has soft-rebooted, this variable is set to the count of soft-reboots.
+        This environment variable is only set for system generators.</para>
```

Comment:
maybe say that this numeric value of this variable will be > 0 always.

Reply:
hmm, I see this as an implementation detail and doesn't really worth tightening the documentation for? Generator impls shall still parse the numeric value before use.